backend/tests/unit/services/prompt/test_prompt_service.py

```
@@ -290,32 +390,97 @@ async def test_update_prompt_success(self, prompt_service, mock_file_ops_service
+                    
+                        result = await prompt_service.update_prompt(
+                            user_id=user_id,
+                            repo_name=repo_name,
+                            file_path=file_path,
+                            prompt_data=update_data
+                        )
         
         assert result is not None
-        assert result.prompt.name == "New Name"
+        # update_prompt returns a tuple (PromptMeta, PRInfo)
+        updated_prompt, pr_info = result
```

Ruff (0.14.1)

Unpacked variable `pr_info` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)